### PR TITLE
Update student notes documentation with tag information

### PR DESCRIPTION
**Pull request by srpearce**
@cahrens, @explorerleslie, @lamagnifica, please review.

@lamagnifica This feature is currently scheduled to release next week. I have doc tasks to move it from the Saudi branch at that time.   

RTD draft: http://draft-student-notes-tags.readthedocs.org/en/latest/SFD_student_notes.html

### CODE REVIEW

File: en_us/students/source/SFD_student_notes.rst [hunk 1]
```diff
@@ -4,80 +4,81 @@
...
+   add. You can also save the highlight for the passage without entering a
+   note or tag.
 
-#. In the field that appears, type the text of your note. Your note can contain
-   as many words as you want.
+   * To enter a note, click **Comments**, and then type the text of your note.
+     Your note can contain as many words as you want.
 
-   .. image:: /Images/SFD_SN_NoteEditor.png
+   * To add one or more tags, click **Add some tags here**, and then type any
```
Comment (srpearce) [inline en_us/students/source/SFD_student_notes.rst:80]: @lamagnifica This UI text is part of the plug-in that we're adding. It needs to stay as-is.

**Comment by cahrens** [comment]
@srpearce Is there a way I can review this as a user will see it (as opposed to having to read the diffs)?

**Comment by srpearce** [comment]
@cahrens I created a draft project here: http://draft-student-notes-tags.readthedocs.org/en/latest/SFD_student_notes.html

File: en_us/students/source/SFD_student_notes.rst [hunk 2]
```diff
@@ -4,80 +4,81 @@
...
+     one word, type multiple words as one word with no spaces, or use hyphens
+     (-) or underscores (_) to separate words in the tag.
+
+    .. image:: /Images/SFD_SN_NoteEditor.png
      :width: 400
      :alt: Note editor open with a student's note
 
-#. After you've finished writing your note, click **Save** or press ENTER. The
-   note editor closes, and the text that you selected remains highlighted.
+   * To highlight a passage without adding a note or tag, click **Save** or
```
Comment (cahrens) [inline en_us/students/source/SFD_student_notes.rst]: This ended up numbering incorrectly (restarting at 1) in the draft.

**Comment by cahrens** [comment]
I just noted one small thing, otherwise :+1:

File: en_us/students/source/SFD_student_notes.rst [hunk 3]
```diff
@@ -4,80 +4,81 @@
...
+top and a pencil icon in the lower right corner. You can see individual notes
+inside the course content, or you can see all of your notes at once on the
+**Notes** tab.
+
+.. image:: /Images/SFD_SN_notesindicators.png
   :width: 500
-  :alt: Image of a course page that includes highlighted text and a student note
+  :alt: Page of course content that includes the Notes tab at the top,
+      highlighted text with an accompanying note, and the pencil icon in the
+      lower right corner
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: I believe we settled on using punctuation in alt tags last week, including periods

File: en_us/students/source/SFD_student_notes.rst [hunk 4]
```diff
@@ -4,80 +4,81 @@
 Student Notes
 ##############################
 
-In some edX courses, you can use the Student Notes feature to highlight and make
-notes about what you read right in the body of the course. You can see
-individual notes inside the course content, or you can see all of your notes at
-once on the **Notes** tab.
+In some edX courses, you can use the Student Notes feature to highlight and
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: I think that student notes should be lower case, or possibly, we should consider dropping "student" altogether and just call it the notes feature

File: en_us/students/source/SFD_student_notes.rst [hunk 5]
```diff
@@ -4,80 +4,81 @@
...
-      in the lower-right corner
+*************************************
+Make a Note or Highlight a Passage
+*************************************
 
-The **Notes** tab lists all the notes that you have made during the course.
-For more information, see :ref:`The Notes Tab`.
+To make a note on a page, follow these steps. 
 
 .. note:: Learners who do not use a mouse can use keyboard shortcuts to create,
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: Learners who do not use a mouse can --> You can also

File: en_us/students/source/SFD_student_notes.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst:80]: do you mean that we have inherited the "**Add some tags here**" wording, and that it cannot (easily) be changed? 'Cause you're right, I absolutely would suggest a different string...

Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst:80]: possibly add that this is optional, and reiterate that tags are used to categorize notes ?

File: en_us/students/source/SFD_student_notes.rst [hunk 6]
```diff
@@ -4,80 +4,81 @@
...
-   note editor closes, and the text that you selected remains highlighted.
+   * To highlight a passage without adding a note or tag, click **Save** or
+     press Enter. When you hover over the highlighted text, the note field
+     contains the words "no comment".
 
-If you want to highlight a passage without making a note, open the note
-editor by following the steps above, and then click **Save** or press ENTER.
-When you hover over the highlighted text, the note field contains the words "no
-comment".
+#. After you've finished writing your note and adding tags, click **Save** or
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: you've finished --> you finish
click --> select
strike "or press Enter"

File: en_us/students/source/SFD_student_notes.rst [hunk 7]
```diff
@@ -95,21 +97,24 @@ one place. You can also search the text of your notes.
...
 
-To search your notes, type a search term in the **Search notes for** box, and
-then either click the magnifying glass icon or press ENTER.
+* A link to the location in the course that contains the highlighted text and
+  your note. To go to that location, click the link under **Noted in**.
+* The time and date when you last edited the note.
+* Any tags that you added.
 
+To search your notes, type a search term in the **Search notes for** field, and
+then either click the magnifying glass icon or press Enter.
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: either click the magnifying glass icon or press Enter. --> select **Search**. (if that's the mouseover tip for the icon).

File: en_us/students/source/SFD_student_notes.rst [hunk 8]
```diff
@@ -95,21 +97,24 @@ one place. You can also search the text of your notes.
...
 
-=======================================
+
+************************************
 Edit or Delete a Note or Highlight
-=======================================
+************************************
 
 To edit a note:
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: To edit a note, follow these steps. 
(similar change needed on line 133)

File: en_us/students/source/SFD_student_notes.rst [hunk 9]
```diff
@@ -132,19 +137,23 @@ To delete a note or highlight:
    an X icon appear in the note field, and then click the X icon to delete the
    note or highlight.
 
-===================
+***************************
 Show or Hide Notes
-===================
+***************************
+
+By default, you can see any notes that you make. However, you can show or hide
```
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: strike "can"

**Comment by lamagnifica** [comment]
just need to apply the style guidelines we've settled on

File: en_us/students/source/SFD_student_notes.rst (hunk 3 above)
Comment (srpearce) [inline en_us/students/source/SFD_student_notes.rst]: I do not recall this conversation - please point me to a record. I notice that this guideline has not been followed in PR 320.

File: en_us/students/source/SFD_student_notes.rst (hunk 4 above)
Comment (srpearce) [inline en_us/students/source/SFD_student_notes.rst]: That's a decision for @explorerleslie. Leslie, do you want to change the name of this advertised feature?

File: en_us/students/source/SFD_student_notes.rst (hunk 1 above)
Comment (srpearce) [inline en_us/students/source/SFD_student_notes.rst:80]: Yes, we inherited this text. I asked about changing it, and it is not possible at this time.

File: en_us/students/source/SFD_student_notes.rst (hunk 2 above)
Comment (srpearce) [inline en_us/students/source/SFD_student_notes.rst]: Thanks - will fix.

File: en_us/students/source/SFD_student_notes.rst (hunk 3 above)
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: May 1st, ~10:50, in Hipchat:
[May-1 10:51 AM] Alison Hodges: question for you: should alt text include punctuation? complete sentences ending with a period, commas and semicolons? or should they be minimally punctuated?
[May-1 10:51 AM] Mark Sadecki: they have a toolbar that will work on HTTP auth pages
[May-1 10:52 AM] Alison Hodges: we don't have too many that aren't public
[May-1 10:52 AM] Alison Hodges: another alt text question is, about how many words is the maximum? I got a lot of "long alt" warnings for the Insights guide
[May-1 10:53 AM] Mark Sadecki: alt text is designed to be concise.  However, if you would use punctuation in that concise statement, then by all means, use it.  Screen readers respect punctuation.  They will pause and may even change pitch if it is a question, etc.
[May-1 10:53 AM] Mark Hoeber: @AlisonHodges  the "private" pages we do have are still "public", just not indexed by search engines
[May-1 10:53 AM] Mark Hoeber: @AlisonHodges  I've been ending alt text with periods

Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: PR 320 predates the conversation in HipChat, May1st, ~10:50:
[May-1 10:51 AM] Alison Hodges: question for you: should alt text include punctuation? complete sentences ending with a period, commas and semicolons? or should they be minimally punctuated?
[May-1 10:51 AM] Mark Sadecki: they have a toolbar that will work on HTTP auth pages
[May-1 10:52 AM] Alison Hodges: we don't have too many that aren't public
[May-1 10:52 AM] Alison Hodges: another alt text question is, about how many words is the maximum? I got a lot of "long alt" warnings for the Insights guide
[May-1 10:53 AM] Mark Sadecki: alt text is designed to be concise.  However, if you would use punctuation in that concise statement, then by all means, use it.  Screen readers respect punctuation.  They will pause and may even change pitch if it is a question, etc.
[May-1 10:53 AM] Mark Hoeber: @AlisonHodges  the "private" pages we do have are still "public", just not indexed by search engines
[May-1 10:53 AM] Mark Hoeber: @AlisonHodges  I've been ending alt text with periods

Comment (srpearce) [inline en_us/students/source/SFD_student_notes.rst]: Thanks. HipChat isn't the best place to document rules; has a story been created to add this to the style guide?

Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: Not yet, it's on my list of things to do today. However, I'm in meetings until 5, so if you get a chance to add the story, that would be great.

**Comment by explorerleslie** [comment]
:+1:

File: en_us/students/source/SFD_student_notes.rst (hunk 7 above)
Comment (srpearce) [inline en_us/students/source/SFD_student_notes.rst]: @explorerleslie The magnifying glass icon for search doesn't have a tooltip - is this a bug?

File: en_us/students/source/SFD_student_notes.rst (hunk 3 above)
Comment (srpearce) [inline en_us/students/source/SFD_student_notes.rst]: Done.

**Comment by srpearce** [comment]
@cahrens @explorerleslie This documentation is ready to go when the feature is released.

File: en_us/students/source/SFD_student_notes.rst (hunk 4 above)
Comment (explorerleslie) [inline en_us/students/source/SFD_student_notes.rst]: I'm definitely open to it. It's more generic and doesn't have student in it.

File: en_us/students/source/SFD_student_notes.rst (hunk 7 above)
Comment (explorerleslie) [inline en_us/students/source/SFD_student_notes.rst]: @srpearce not sure. Maybe @cptvitamin or @marcotuts can comment.

File: en_us/students/source/SFD_student_notes.rst (hunk 4 above)
Comment (lamagnifica) [inline en_us/students/source/SFD_student_notes.rst]: My thinking is, this is the "student facing" documentation, so calling it "Student Notes" seems overly obvious. 
"In some edX courses, you can highlight text and make notes about what you read right in the body of the course."

File: en_us/students/source/SFD_student_notes.rst (hunk 7 above)
Comment (cptvitamin) [inline en_us/students/source/SFD_student_notes.rst]: Good point @srpearce. This control is a standard `<button>` element with a screen reader label.  It is accessible to all as long as the icon visually displays. there is an teeny tiny edge case that may cause the icon to not display, which would be problematic. While the magnifying glass paradigm is well known, the previously noted edge case, as well as the condition where someone wasn't familiar with the paradigm, would be remedied with a tooltip.  NOTE: The current edX tooltip system does not fire on keyboard focus, only firing on mouse hover (needs to be fixed).   I would be in favor of adding a tooltip here with the standard low-priority justification of "It doesn't hurt".
cc @marcotuts @explorerleslie

**Comment by srpearce** [comment]
Superseded by PR 346 (https://github.com/edx/edx-documentation/pull/346)

**Pull request closed without merging**